### ISSUE TITLE
Handling of entryCondition vs prerequisite

### ISSUE THREAD

**Issue by siuc-nate** [CredentialEngine/Schema-Development#831]
We currently have these properties:
- [entryCondition](https://credreg.net/ctdl/terms/entryCondition) 
  - used for credentials, assessments, learning opportunities, etc.,
  - definition: "Prerequisites for entry into a credentialing program, a learning opportunity or an assessment including transcripts, records of previous experience, and lower-level learning opportunities."
- [prerequisite](https://credreg.net/ctdl/terms/prerequisite)
  - used for pathway components
  - definition: "Resource that is required as a prior condition to this resource."

Both of these properties (regardless of the subtle differences in their definitions) serve the same fundamental purpose: "you must have done x before beginning to do y".

In today's meeting, we discussed changing the domain of prerequisite to make it match the domain of entryCondition - but then we will have two largely (if not entirely, in terms of actual usage) identical properties.

The purpose of this issue is to discuss the matter further and determine the best course of action.

**Comment by siuc-nate** [comment]
It may be that we can emphasize entryCondition's notion of "thing you must have" (e.g. transcript) and emphasize prerequisite's notion of "thing you must have done" (e.g. passed a test) to better differentiate the two.

**Comment by philbarker** [comment]
Sorry I missed the discussion yesterday. 

Was [`requires`](https://purl.org/ctdl/terms/requires) "Requirement or set of requirements for this resource" also discussed?

One factor that we should consider is that these terms all have certain nuanced meanings when used in education contexts. Entry Conditions tend to be what is required to join from outside, prerequisites tend to be used as short-hand for education or competences required, and are often applied within whatever structure is being described. For example: joining a degree program would come with entry conditions;  progression along a pathway for that program is not "entry" and so `requires`/`prerequisite` is more natural than `entryCondition`; and course A may be a prerequisite for course C within the program.

Also, educational prerequisites can usefully be scoped as being about required skills, knowledge, attainment etc. Possession of your own horse might be a requirement, but it would be useful to distinguish this type of requirement from prerequisite skills such as "must be able to ride a horse" or "must have successfully completed our course on how to ride a horse" or "must have a horse riding certificate".

FWIW, I think it a shame that `prerequisite` is used on pathway components not on learning opportunities themselves.

In summary: consider whether we are talking about entry, progression or graduation, and consider the range (what is required) as well as the domain. We're not starting from scratch, so where that lands will depend on existing usage as much as beign able to draw logical and useful distinctions.

**Comment by siuc-nate** [comment]
See also #771

**Comment by jeannekitchens** [comment]
This issue was moved to #846.
